camelConnector/src/test/java/io/vantiq/extsrc/camelconn/connector/VantiqConfigurationTest.java

```diff
@@ -358,6 +400,123 @@ public void testComponentInitConfigurationWithPropertyValues() {
+        performConfigTest(name.getMethodName(), vantiqRoute, "yaml",
+                          null, null,
+                          (CamelContext runnerContext) -> {
+                              TriFunction<CamelContext, Map<String, Object>, Map<String, Object>, Boolean> verifyOperation =
+                                      defineVerifyHeaders(1, "direct:vantiqStart",
+                                                          vantiqEpUri);
+                              for (int i = 0; i < 10; i++) {
+                                  String h1Val = "h1Value-" + i;
+                                  String h2Val = "h2Value-" + i;
+                                  String h3Val = "h3Value-" + i;
+                                  assert verifyOperation.apply(runnerContext,
+                                                               Map.of("headers", Map.of("hdr1", h1Val,
+                                                                                        "hdr2", h2Val,
+                                                                                        "hdr3", h3Val),
+                                                                      "message", Map.of("bodyPart", "arm")),
+                                                               Map.of("hdr1", h1Val,
+                                                                      "hdr2", h2Val,
+                                                                      "hdr3", h3Val,
+                                                                      "otherHdr1", h1Val,
+                                                                      "otherHdr2", h2Val,
+                                                                      "otherHdr3", h3Val));
+                              }
+                          }, null, dupBeanName, hdrDups, oRides);
+    }
+    
+    /**
+     * Create a callable that the test method will call.
+     *
+     * In this case, the callable "sends"
+     * message to the route which, in turn, makes a call to return some data.  We verify that the expected
+     * results are presented.
+     *
+     * In this case, our route uses the dynamically loaded component to make a call.
+     * @return TriFunction<CamelContext, String, Object, Boolean>
+     */
+    public TriFunction<CamelContext, Map<String, Object>, Map<String, Object>, Boolean> defineVerifyHeaders(int msgCount, String startEp,
+                                                                                    String endEp) {
+        TriFunction<CamelContext, Map<String, Object>, Map<String, Object>, Boolean> verifyOperation =
+                (context, query, answerMap) -> {
+            boolean worked = true;
+            String routeId = context.getRoutes().get(0).getId();
+            ProducerTemplate template = null;
+            try {
+                if (context.isStopped()) {
+                    log.error("At test start, context {} is stopped", context.getName());
```

Comment:
I assume that the intent is to let processing to continue despite logging an error?